docs/idle-crafting.rst

```diff
@@ -0,0 +1,56 @@
+idle-crafting
+=============
+
+.. dfhack-tool::
+    :summary: Allow dwarfs to automatically satisfy their need to craft objects.
+    :tags: fort gameplay
+
+This script allows dwarves to automatically satisfy their crafting needs. The
+script is configured through an overlay that is added to the main page of
+craftsdwarf's workshops, as described below.
+
+Usage
+-----
+
+::
+
+    idle-crafting status
+    idle-crafting disable
+    disable idle-crafting
+
+The ``status`` command prints statistics about the status of the tool and the
+satisfaction of "craft item" needs in your fort. Both variants of the
+``disable`` command disallow idle crafting at all workshops and disable the
+tool.
+
+Examples
+--------
+
+``idle-crafting -t 10000,1000,500 status``
+    Reset all thresholds to defaults and print status information.
+
+Options
+-------
+
+``-t <number list>``, ``--thresholds <number list>``
+    Sets the threshold(s) for the "craft item" need (i.e. the negated
+    ``focus_level``) at which the tool starts to generate crafting jobs for a
+    given unit. Units meeting earlier (higher) thresholds will be
+    prioritized. Defaults to ``10000,1000,500``.
```

Comment:
I suggest accepting the list of numbers in any order and then sorting/deduping to canonicalize. In particular, for the documentation I suspect writing examples in ascending order (`500,1000,10000`) would be more understandable.

Reply:
I went with sorting only, duplicate threshold will just lead to empty categories, which should not have any effect whatsoever.